### ISSUE TITLE
Just curious. How does this work on windows 10? 

### ISSUE THREAD

**Issue by DOOMMARINE117** [felixrieseberg/windows95#175]
⚠️ Thank you for reporting an issue!

Before we go any further, understand that I probably won't be able to fullfil feature requests.
Feel free to report what feature you'd love to see, just don't get angry when I don't have
time to implement it 🙇‍♂️

I will however _gladly_ help you make a pull request if you're willing to play with Javascript!

im curious on how these software programs work for windows 10 of windows 95? 

will floppy disc or full version of dos cd games will work on the app? 

or is it a software to install or is it a website?

**Comment by Lethdev2019** [comment]
## overview
both floppy disks and the full versions should work (if a bit slow).

note 32-bit may not work and 64-bit will most definitely not work.
this is because windows 95 runs on the 16-bit version of FAT otherwise known as FAT16 on MSDOS 5 (i think as MSDOS 6 and above was developed **after** windows 95.

in order to insert floppy disks you need a .img file, you then need to click settings and insert the "floppy disk".
if you have a .iso, just change the extension to .img. (according to various articles)

## how does it work
The emulator runs in a `.html` file (webpage document) which is powered by multiple `.js` scripts which are buried in `src` or `dist`.
The application the webpage runs in is Electronjs which you can find at https://electronjs.org which uses nodejs as it's backend.

Now please do take into account that scripts in the .html file has nodejs integration meaning it can perform file system operations like saving and loading files.

I do not know all the details except that the emulator originated from https://copy.sh/v86.

hope this helps.

**Comment by DOOMMARINE117** [comment]
> 
> ## overview
> 
> both floppy disks and the full versions should work (if a bit slow).
> 
> note 32-bit may not work and 64-bit will most definitely not work.
> this is because windows 95 runs on the 16-bit version of FAT otherwise known as FAT16 on MSDOS 5 (i think as MSDOS 6 and above was developed **after** windows 95.
> 
> in order to insert floppy disks you need a .img file, you then need to click settings and insert the "floppy disk".
> if you have a .iso, just change the extension to .img. (according to various articles)
> ## how does it work
> 
> The emulator runs in a `.html` file (webpage document) which is powered by multiple `.js` scripts which are buried in `src` or `dist`.
> The application the webpage runs in is Electronjs which you can find at https://electronjs.org which uses nodejs as it's backend.
> 
> Now please do take into account that scripts in the .html file has nodejs integration meaning it can perform file system operations like saving and loading files.
> 
> I do not know all the details except that the emulator originated from https://copy.sh/v86.
> 
> hope this helps.

so sounds like this emulator won't work on my 64 bit? 

interesting.

**Comment by Lethdev2019** [comment]
no, the emulator will run on a 64 bit computer, as 64 bit computers can run 32-bit and 64-bit programs (thanks to the system folder "wow64" or "Windows on Windows")

what i am saying is the **emulator** cannot run 32 bit and above programs.

**Comment by DOOMMARINE117** [comment]
> 
> 
> no, the emulator will run on a 64 bit computer, as 64 bit computers can run 32-bit and 64-bit programs (thanks to the system folder "wow64" or "Windows on Windows")
> 
> what i am saying is the **emulator** cannot run 32 bit and above programs.
> you have to insert a 16-bit or 8-bit image into the emulator.

i see. gotcha.

when using the emulator. i can install any program that will requrie this image to 64 bit program? like say: a dos game?

**Comment by Lethdev2019** [comment]
dos games are only 16-bit.
the programs that you add must be designed for MS-DOS or win9.x meaning that you cannot run 32-bit or 64-bit programs or if in the desktop environment 32-bit programs **might** work.

**Comment by DOOMMARINE117** [comment]
> 
> 
> dos games are only 16-bit.
> the programs that you add must be designed for MS-DOS or win9.x meaning that you cannot run 32-bit or 64-bit programs or if in the desktop environment 32-bit programs **might** work.
> 
> windows 95 is weirdo, ok? it is a hybrid system, booting of a 16-bit MS-DOS bootloader but it is 32-bits.
> 
> > how did Microsoft pull that one off.

gotcha. 

that is very interesting actually.

i see,m thanks for your help. 

i hope you don't mind all the questions. 

its quite interesting stuff.

**Comment by felixrieseberg** [comment]
It's a desktop app you install (not a website): a JavaScript/WebAssembly PC emulator (v86) with a real Windows 95 disk image, wrapped in Electron. And yes — floppy images (`.img`) and CD images (`.iso`) mount right in the app now, so era-appropriate games generally work. Closing.
